### ISSUE TITLE
QA - Images Need Alt Text field in CMS

### ISSUE THREAD

**Issue by data-sync-user** [MozillaFoundation/foundation.mozilla.org#14302]
**Context:** Any component that allows an image to be included needs to also have an editable alt text field for accessibility reasons, unless the image is purely decorative. Right now it appears hero images and portrait cards don’t have these fields, but there may be others.

Pages this impacts: Make, Inspire, Mobilize, What We do, Meet Mozilla

**How to replicate:** Log into CMS, see[ pages listed here|https://www.mozillafoundation.org/cms/pages/31532/], click into ‘[Inspire'](https://www.mozillafoundation.org/cms/pages/31568/edit/), see hero section component and hero image, see alt text field is missing.

**Request:** Add this field to any components that use imagery (make required unless the image is purely decorative)

!Screenshot 2025-07-22 at 9.08.04 AM.png|width=825,height=106,alt="Screenshot 2025-07-22 at 9.08.04 AM.png"!

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/TP1-2990)
┆Attachments: <a href="https://mozilla-hub.atlassian.net/rest/api/2/attachment/content/90644">Screenshot 2025-07-22 at 9.08.04 AM.png</a>
